apps/dashboard/src/app/api/auth/get-auth-token/route.ts

```diff
@@ -0,0 +1,58 @@
+import { COOKIE_PREFIX_TOKEN } from "@/constants/cookie";
+import { cookies } from "next/headers";
+import { type NextRequest, NextResponse } from "next/server";
+import { getAddress } from "thirdweb/utils";
+
+const THIRDWEB_API_HOST =
+  process.env.NEXT_PUBLIC_THIRDWEB_API_HOST || "https://api.thirdweb.com";
+
+export type GetAuthTokenResponse = {
+  jwt: string | undefined;
+};
```

Comment:
respond with `string | null`, `undefined` is not serializable

Reply:
change to `null` 👍